packages/js/src/shared-admin/components/premium-upsell-list.js

```diff
@@ -59,28 +59,22 @@ export const PremiumUpsellList = ( { premiumLink, premiumUpsellConfig = {}, isPr
 			</div> }
 			<div className="yst-p-6 yst-flex yst-flex-col">
 				<div className="yst-flex yst-items-center">
-					{ isWooCommerceActive
-						? <>
-							<Title as="h2" size="4" className={ `yst-text-xl ${ isWooCommerceActive ? "yst-text-woo-light" : "yst-text-primary-500 " }` }>
-								{ sprintf(
+					<>
+						<Title as="h2" size="4" className={ `yst-text-xl yst-font-semibold ${ isWooCommerceActive ? "yst-text-woo-light" : "yst-text-primary-500 " }` }>
+							{ isWooCommerceActive
+								? sprintf(
 									/* translators: %s expands to "Yoast SEO" Premium */
```

Comment:
```suggestion
									/* translators: %s expands to "Yoast WooCommerce SEO" */
```

Reply:
Also from this page, is it expected that now we always use the crown icon also for Yoast WooCommerce SEO ads?
If yes, this import needs to be removed from this file: 
```
import { ReactComponent as TrolleyIcon } from "../../../images/icon-trolley.svg";
```